dev/pages/Card.vue

```diff
@@ -29,7 +29,7 @@ const radioModel = ref('')
         </w-card>
         <w-card :selected="selected">
           <img class="h-128 w-full object-cover" src="https://source.unsplash.com/random/400x402" />
-          <p class="absolute top-12 left-12 bg-aqua-200 text-aqua-900 p-4 rounded-4 text-12">Ukens bolig</p>
+          <w-badge variant="info" position="top-left">Ukens bolig</w-badge>
           <div class="p-16">
             <p class="text-12 text-gray-300">DNB Eiendom</p>
```

Comment:
Some unavailable classes there as well `text-gray-300`, but maybe that's cleanup for another time

Reply:
Seems like a good opportunity to fix that as well... 👍 😄  Will do.